pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -36,14 +36,14 @@
 log.addHandler(logging.NullHandler())
 
 
-def _sanitize_source(source):
+def sanitize_source(source):
```

Comment:
Do you expect your users to call this method directly? If not, why remove the leading underscore that indicates a "private" method?

Reply:
Exactly. I needed to call the `sanitize_source` method in one of my other projects directly. Some queries on the scope return the source name and I might want to compare it with the input source name.

Reply:
OK, thanks!